Geometry/GEMGeometryBuilder/src/ME0GeometryBuilderFromDDD.cc

```diff
@@ -1,3 +1,12 @@
+/*
+//\class ME0GeometryBuilder
```

Comment:
Now is the time to change this name, `ME0GeometryBuilderFromDDD`. "DDD" means the old DD, but you are now using this name for both the old DD and DD4hep. Please change `ME0GeometryBuilderFromDDD` to `ME0GeometryBuilder` or some similar more generic name.

Reply:
@cvuosalo : done